tutorial/source/tracking_1d_em.ipynb

```diff
@@ -217,12 +219,13 @@
     "        next_id += 1\n",
     "        # Merging\n",
     "        states_loc[k] = (states_loc.pop(i) + states_loc.pop(j)) / 2\n",
-    "        merge_fn = lambda a,b: (a.pow(2) + b.pow(2)) / (a + b) #contra-mean\n",
+    "        #merge_fn = lambda a,b: (a.pow(2) + b.pow(2)) / (a + b) #contra-mean\n",
+    "        merge_fn = lambda a,b: (a.pow(1/2) + b.pow(1/2)) / (a + b) #inverse contra-mean\n",
     "        #merge_fn = lambda a,b: 2 / (1 / a + 1 / b) # harmonic mean\n",
     "        #merge_fn = lambda a,b: a + b # sum\n",
     "        exist_dist_probs[k] = merge_fn(exist_dist_probs.pop(i),exist_dist_probs.pop(j))\n",
-    "        exist_dist_probs[k].clamp_(min=0.0, max=1.0)\n",
     "        assign_dist_probs[k] = (assign_dist_probs.pop(i) + assign_dist_probs.pop(j))\n",
+    "        exist_dist_probs[k].clamp_(min=0.0, max=1.0)\n",
```

Comment:
I think it's cleaner to avoid manipulating `exists_probs` and `assign_probs` in merge.